dotnetv3/Lambda/SetuplambdaRoleExample/SetuplambdaRoleExample/SetupLambdaRole.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace SetuplambdaRoleExample
+{
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.IdentityManagement;
+    using Amazon.IdentityManagement.Model;
+
+    /// <summary>
+    /// Creates an Amazon Identity Management Role to attach to an Amazon
```

Comment:
AWS Identity and Access Management (IAM)

Reply:
Fixed